src/components/ColorPickerInput/ColorPickerInput.scss

```diff
@@ -15,6 +15,8 @@ $block: '.#{variables.$ns}color-picker';
         width: 100%;
         height: 0;
         opacity: 0;
-        border: 1px solid transparent;
+        padding: 0;
+        margin: 0;
+        border: 0;
```

Comment:
are u check result in safari and firefox? 
that was a really hard to fix safari problems (if whole height of block become less then 5px - color modal go to different place)

Reply:
I checked it in safari and firefox. Everything is OK.